cuqi/samples/_samples.py

```diff
@@ -190,6 +192,16 @@ def geometry(self):
             self._geometry = _DefaultGeometry(grid=np.prod(self.samples.shape[:-1]))
         return self._geometry
 
+    @property
+    def _funvals(self):
```

Comment:
Perhaps we should expose this property publicly? Perhaps make it a method and call it compute_funvals().

Reply:
Great, just did that.